sdk/js-query/src/mock/index.ts

```diff
@@ -348,6 +352,80 @@ export class QueryProxyMock {
             )
           )
         );
+      } else if (type === ChainQueryType.SolanaAccount) {
+        const query = perChainRequest.query as SolanaAccountQueryRequest;
+        // Validate the request.
+        if (query.commitment !== "finalized") {
+          throw new Error(
+            `Invalid commitment in sol_account query request, must be "finalized"`
+          );
+        }
+        if (
+          query.dataSliceLength === BigInt(0) &&
+          query.dataSliceOffset !== BigInt(0)
+        ) {
+          throw new Error(
+            `data slice offset may not be set if data slice length is zero`
+          );
+        }
+        if (query.accounts.length <= 0) {
+          throw new Error(`does not contain any account entries`);
+        }
+        if (query.accounts.length > 255) {
+          throw new Error(`too many account entries`);
+        }
+
+        let accounts: string[] = [];
+        query.accounts.forEach((acct) => {
+          if (acct.length != 32) {
+            throw new Error(`invalid account length`);
+          }
+          accounts.push(base58.encode(acct));
+        });
+
+        const response = await axios.post(rpc, {
+          jsonrpc: "2.0",
+          id: 1,
+          method: "getMultipleAccounts",
+          params: [accounts, { commitment: query.commitment }],
+        });
+
+        const slotNumber = response.data.result.context.slot;
+        let results: SolanaAccountResult[] = [];
+        response.data.result.value.forEach((v: Object) => {
+          const val = v as SolanaAccountResult;
```

Comment:
This `as` is a bit odd, since it seems like `data` below doesn't line up. it's a Uint8Array on that type, but below you're indexing it at `[0]`. Might be worth a different type defined in this mock and used on `axios.post`, like `axios.post<SolanaMultipleAccountsResponse>(...)`